index.html

```diff
@@ -505,7 +505,7 @@ <h5 class="">Team Member 1</h5>
           <div class="col-lg-12 col-md-12">
             <!-- START BLOG HEADING -->
             <div class="heading">
-              <h2 class="wow fadeInLeftBig">Latest News From Blog</h2>
+              <h2 class="wow fadeInLeftBig">Projects</h2>

```

Comment:
This correction will be done in the third issue. One of the second year students is already working on that. So, I can't merge it.